### PR TITLE
Add Staking Banner

### PR DESCRIPTION
**Pull request by HrikB**
Added the staking banner from the figma design.

Fixed some responsiveness issues with the OGV price chart on mobile.

Also addressed error from staging: 
`Error: Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations  react/jsx-no-target-blank`

### CODE REVIEW

**Comment by HrikB** [comment]
@shahthepro Regarding the tailwind issue, I was able to reproduce it on a new laptop. Project still isn't able to pick up tailwind classes without it being included in the `globals.css` file. Is there something else I need to install perhaps?

File: pages/ogv-dashboard.tsx [hunk 1]
```diff
@@ -724,13 +762,44 @@ const OgvDashboard = ({
         </div>
       </section>
 
+      {/* OGV Staking */}
+      <section className="bg-[#141519] px-8 md:px-16 lg:px-[8.375rem]">
```
Comment (shahthepro) [inline pages/ogv-dashboard.tsx:766]: nit: Let's not use arbitary values for the colors

**Review by shahthepro — APPROVED** [review]
LGTM

File: pages/ogv-dashboard.tsx (hunk 1 above)
Comment (HrikB) [reply pages/ogv-dashboard.tsx:766]: Done!

**Pull request merged**